README.md

```diff
@@ -30,3 +30,5 @@ If you're curious about the tools, stacks and technologies we use, get ready for
 - apply directly via [our hiring page](http://carrot.is/hiring)
 - [file an issue](https://github.com/carrot/we-are-hiring/issues) with a question you might have
 - fork this repository and PR in your resume/application (or just something we might find interesting)
+
+Last Updated: 04-10-2015 08:27 AM
```

Comment:
: ) 

the script is going to make a throwaway commit on a throwaway branch every X hours (on a cron) to keep the repo relatively at the top of our http://github.com/carrot listing